modules/SofaConstraint/src/SofaConstraint/FreeMotionAnimationLoop.cpp

```diff
@@ -286,11 +259,12 @@ void FreeMotionAnimationLoop::step(const sofa::core::ExecParams* params, SReal d
         }
     }
 
-    if (!SOFA_NO_UPDATE_BBOX)
+#if SOFA_NO_UPDATE_BBOX == 0
```

Comment:
So it seems to you are doing the inverse move from what I consider the best way to do...could you elaborate a bit why ?